src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+4.1.0
+-----
+
+* added `IncompleteInputDataException` new exception for deserialization failure
+  of objects that needs data insertion in constructor
```

Comment:
Is it only when the constructor requires specific arguments? If yes then something like `MissingConstructorArgumentsException` would make more sense. If not, then it needs to be clarified here :)